wgpu/Cargo.toml

```diff
@@ -197,19 +148,47 @@ serde = { workspace = true, features = ["default", "derive"], optional = true }
 smallvec.workspace = true
 static_assertions.workspace = true
 
-[dependencies.naga]
-workspace = true
-optional = true
+[target.'cfg(any(target_arch = "wasm32", not(target_os = "emscripten")))'.dependencies]
+js-sys = { workspace = true, features = ["default"] }
 
-[build-dependencies]
-cfg_aliases.workspace = true
+########################################
+# Target Specific Feature Dependencies #
+########################################
+
+# Windows
+[target.'cfg(windows)'.dependencies]
+wgpu-core = { workspace = true, features = [
+    "raw-window-handle",
+    "vulkan",
+    "gles",
+] }
+wgpu-hal = { workspace = true, features = ["renderdoc"] }
+
+# Apple Platforms
+[target.'cfg(target_vendor = "apple")'.dependencies]
+wgpu-core = { workspace = true, features = ["raw-window-handle"] }
+wgpu-hal = { workspace = true, features = [] }
+
+# Linux + Android
+[target.'cfg(any(target_os = "linux", target_os = "android"))'.dependencies]
+wgpu-core = { workspace = true, features = [
+    "raw-window-handle",
+    "vulkan",
+    "gles",
+] }
+wgpu-hal = { workspace = true, features = ["renderdoc"] }
 
-# used to test all the example shaders
-[dev-dependencies.naga]
-workspace = true
-features = ["wgsl-in"]
```

Comment:
The `"wgsl-in"` feature seems to have disappeared, wasn't it needed?

Reply:
No, this was a very old thing from before we moved the test suite into its own crate. This does nothing right now, as wgpu has no examples or tests.